haiku-driver.groovy

```diff
@@ -26,8 +31,77 @@ def updated() {
     log.debug "updated"
 }
 
+def setFanDirection(String direction) {
+    sendCommand("FAN", "DIR", "SET;${direction}")
+}
+
+def reverseFan() {
+    if (device.currentValue("fanDirection") == "FWD") {
+        setFanDirection("REV")
+    } else {
+        setFanDirection("FWD")
+    }
+}
+
+def refresh() {
+    sendCommand("LIGHT", "LEVEL", "GET;ACTUAL")
+    sendCommand("FAN", "DIR", "GET")
+}
+
 def parse(String description) {
     log.debug "parse description: ${description}"
+    def map = parseLanMessage(description)
+    def bytes = map["payload"].decodeHex()
+    def response = new String(bytes)
+    log.debug "parse response: ${response}"
+    def values = response[1..-2].split(';')
+    //setName(values[0])
```

Comment:
Unused?

Reply:
Oops, that snuck into a commit. Removed.